homeassistant/const.py

```diff
@@ -303,6 +303,9 @@
 # Temperature attribute
 ATTR_TEMPERATURE = 'temperature'
 
+# Current Temperature attribute
```

Comment:
Please don't add this to main const but import it from climate component.

Reply:
I've removed this from the main constants and I'm now importing it from the climate component.  Thanks.